app/models/user.rb

```diff
@@ -30,6 +31,22 @@ def online?
     last_sign_out_at.present? ? last_sign_out_at > Time.current : false
   end
 
+  def self.from_omniauth(access_token)
```

Comment:
would like this to be a separate class that controls auth

create app/services/auth/omni_google.rb
class will receive access_token and have method user that will either searc or create user

Reply:
Currently User model will fail single responsibility principle
as user model has nothing to do with omniauth